EKF/imu_down_sampler.hpp

```diff
@@ -50,9 +50,9 @@ class ImuDownSampler
 	explicit ImuDownSampler(float target_dt_sec);
 	~ImuDownSampler() = default;
 
-	bool update(const imuSample &imu_sample_new);
+	bool update(const imuSample& imu_sample_new);
```

Comment:
```suggestion
	bool update(const imuSample &imu_sample_new);
```
I think we're usually using this convention, no?

Reply:
Yes, but apparently I have bad muscle memory. I usually let the tool handle it.